File: Clase3/Python/Operadores_logicos.py

nombre="luis"
edad= 20

print("Hola", nombre)
print(f"Hola {nombre}, tienes {edad} años")  # f-string

mayor_de_edad = edad >= 18
es_estudiante = True

if mayor_de_edad and es_estudiante:
    print("Es mayor de edad y estudiante")
    print("ya toca el timbre")

File: Clase3/Java/Operadores_logicos.java
public class Operadores_logicos {
    public static void main(String[] args) {

        String nombre = "alan";
        int edad = 20;
        System.out.println("Hola " + nombre);
        System.out.printf("Hola %s, tienes %d años\n", nombre, edad);

        boolean mayorDeEdad = edad >= 18;
        boolean esEstudiante = true;
        
        if (mayorDeEdad && esEstudiante) {
            System.out.println("Es mayor de edad y estudiante");
        }
    }
}

